onecodex/viz/_metadata.py

```diff
@@ -168,20 +167,31 @@ def plot_metadata(
 
             if title:
                 chart = chart.properties(title=title)
-        elif plot_type == "boxplot":
+        elif plot_type == PlotType.BoxPlot:
             if sort_x:
                 raise OneCodexException("Must not specify sort_x when plot_type is boxplot")
 
-            chart = boxplot(
-                df,
-                magic_fields[haxis],
-                magic_fields[vaxis],
-                category_type=category_type,
-                title=title,
-                xlabel=xlabel,
-                ylabel=ylabel,
+            chart = (
+                alt.Chart(df)
+                .mark_boxplot(size=35)
+                .encode(
+                    x=alt.X(magic_fields[haxis], axis=alt.Axis(title=xlabel)),
+                    y=alt.Y(magic_fields[vaxis], axis=alt.Axis(title=ylabel)),
+                )
             )
 
+        props = {}
+
+        if title:
+            props["title"] = title
+        if width:
+            props["width"] = width
+        if height:
+            props["height"] = height
+
+        if props:
+            chart = chart.properties(**props)
```

Comment:
this code is repeated a few times, something updating the chart title too. Can it be abstracted? Not a blocker

Reply:
Simplified.